sdk/src/contexts/solana/context.ts

```diff
@@ -1176,4 +1187,25 @@ export class SolanaContext<
       chain: 'solana',
     };
   }
+
+  async getPriorityFeeIx(address: string) {
+    let unitPrice = 10_000; // use minimum of 10,000 microlmaports
+
+    const recentFees = await this.connection?.getRecentPrioritizationFees({
+      lockedWritableAccounts: [new PublicKey(address)],
+    });
+
+    if (recentFees) {
+      const maxFee = Math.max(
+        ...recentFees.map(({ prioritizationFee }) => prioritizationFee),
+      );
+      unitPrice = Math.max(unitPrice, maxFee * 10);
```

Comment:
Would be good to document why 10x chosen here. Also what happens if there are multiple Connect transactions submitted within a short period of time, could the later ones detect earlier ones resulting in 10*10*10...

Reply:
yeah I had this same worry. I was just trying this to see if it helped with timeouts, but it wasn't.